exporter/googlecloudexporter/go.mod

```diff
@@ -25,7 +25,7 @@ require (
 	cloud.google.com/go v0.94.1 // indirect
 	cloud.google.com/go/monitoring v0.1.0 // indirect
 	cloud.google.com/go/trace v0.1.0 // indirect
-	github.com/aws/aws-sdk-go v1.37.0 // indirect
+	github.com/aws/aws-sdk-go v1.40.56 // indirect
```

Comment:
Note for posterity/other reviewers: this PR also bumps `indirect` dependencies (which should be fine if semver is followed)